pkg/server/config.go

```diff
@@ -160,5 +161,21 @@ func (cfg *Config) ToFlags(set *cli.FlagSet) *cli.FlagSet {
 		Usage:  `The number of minutes to cache configuration files before retrieving fresh ones. Defaults to 15 minutes.`,
 	})
 
+	f.DurationVar(&cli.DurationVar{
+		Name:    "jwks-cache-duration",
+		Target:  &cfg.JWKSCacheDuration,
+		EnvVar:  "JWKS_CACHE_DURATION",
+		Usage:   `The duration to cache the JWKS for an OIDC token issuer. Defaults to 4 hours.`,
+		Default: 4 * time.Hour,
+	})
+
+	f.StringSliceVar(&cli.StringSliceVar{
+		Name:    "issuer-allowlist",
+		Target:  &cfg.IssuerAllowlist,
+		EnvVar:  "ISSUER_ALLOWLIST",
+		Usage:   `The list of OIDC token issuers that GitHub Token Minter will accept. Format is a comma-separated list of URLs. Defaults to GitHub and Google issuers.`,
```

Comment:
```suggestion
		Usage:   `The list of OIDC token issuers that GitHub Token Minter will accept. Format is a comma-separated list of URLs.`,
```

And technically this accepts a comma-separated list and/or the flag can be specified multiple times.